tests/test_modeling_auto.py

```diff
@@ -100,3 +101,12 @@ def test_from_pretrained_identifier(self):
         logging.basicConfig(level=logging.INFO)
         model = AutoModelWithLMHead.from_pretrained(SMALL_MODEL_IDENTIFIER)
         self.assertIsInstance(model, BertForMaskedLM)
+        self.assertEqual(model.num_parameters(), 14830)
+        self.assertEqual(model.num_parameters(only_trainable=True), 14830)
+
+    def test_from_identifier_from_model_type(self):
```

Comment:
Same comment as in `test_configuration_auto` here, we should have checks on the common failure cases of the dict mapping.